web/crux-ui/src/components/products/versions/version-view-list.tsx

```diff
@@ -85,13 +82,11 @@ const VersionViewList = (props: VersionViewListProps) => {
         <Image className="cursor-pointer" src="/trash-can.svg" width={24} height={24} onClick={() => onDelete(item)} />
       </div>
       <div className="inline-block">
-        <Image
-          className="cursor-pointer"
-          src="/settings.svg"
-          width={24}
-          height={24}
-          onClick={() => onImageSettings(item)}
-        />
+        <Link href={imageConfigUrl(state.productId, state.versionId, item.id)}>
```

Comment:
this needs passHref i think

Reply:
Afaik no, passHRef is only needed if the child is a custom component that wraps an <a> tag, but we have the <a> as a direct child, so we don't need to force it.